packages/cache/src/cache.ts

```diff
@@ -108,12 +106,12 @@ export async function restoreCache(
 /**
  * Restores cache using the legacy Cache Service
  *
- * @param paths
- * @param primaryKey
- * @param restoreKeys
- * @param options
- * @param enableCrossOsArchive
- * @returns
+ * @param paths a list of file paths to restore from the cache
+ * @param primaryKey an explicit key for restoring the cache
```

Comment:
FYI our documentation is incorrect for the primary key - it uses prefix matching as well per the current server side implementation

https://github.com/actions/cache/issues/1433#issuecomment-2259294936